### PR TITLE
Document `cfg.fortunes.file`

### PR DESCRIPTION
**Pull request by officialcjunior**
Document `cfg.fortunes.file`, which is the var that helps you choose the type of fortunes.

### CODE REVIEW

File: src/configuration/evars.md [hunk 1]
```diff
@@ -246,6 +246,10 @@ A boolen variable to show UTF-8 characters instead of ANSI.
 
 Enables or disables "fortune" messages displayed at each rizin start.
 
+### cfg.fortunes.file
+
+Rizin has two types for fortunes: tips and fun. Fortunes of the type 'tips' are general tips to help you use Rizin better, whereas the other one prints some lighthearted jokes. You can choose which type of fortune to display, using this variable.
```
Comment (wargio) [inline src/configuration/evars.md:251]: Actually you can also specify a path (relative or absolute) to a custom fortune file.

**Review by wargio — CHANGES_REQUESTED** [review]

File: src/configuration/evars.md (hunk 1 above)
Comment (wargio) [reply src/configuration/evars.md:251]: like `e cfg.fortunes.file /path/to/my/fortunes.txt`

Comment (ITAYC0HEN) [reply src/configuration/evars.md:251]: Need to update inline docs in Rizin itself as well, because when I read the help message it wasn't clear

Comment (officialcjunior) [reply src/configuration/evars.md:251]: Alright, I'll add that at the end in a second.
I can help with the inline docs too, if everyone else has other important stuff to do.

**Review by wargio — APPROVED** [review]

**Comment by wargio** [comment]
@ITAYC0HEN if you feel that this definition is ok, then merge this.
@officialcjunior awesome job (as always)

File: src/configuration/evars.md (hunk 1 above)
Comment (ITAYC0HEN) [reply src/configuration/evars.md:251]: Another thing, in r2 you could mentioned multiple fortune files, e.g "fun,tips". How does it work now? If it is still like it, need to be mentioned and demonstrated

Comment (wargio) [reply src/configuration/evars.md:251]: right now there is no multiple file fortunes

File: src/configuration/evars.md [hunk 2]
```diff
@@ -246,6 +246,16 @@ A boolen variable to show UTF-8 characters instead of ANSI.
...
+
+Rizin also supports custom fortunes. You can save your fortunes in a file and provide the relative or absolute path to the same variable, to have them displayed at the startup.
+
+```
+[0x00000000]> e cfg.fortunes.file = /path/to/my/fortunes.txt
```
Comment (wargio) [inline src/configuration/evars.md:256]: Probably this is my mistake, but actually there must not be any space between the `=`.
so it should be: `e cfg.fortunes.file=/path/to/my/fortunes.txt`

**Review by wargio — CHANGES_REQUESTED** [review]

**Review by ITAYC0HEN — APPROVED** [review]

**Pull request merged**